doc/operation-and-maintenance/Mongoose-metrics.md

```diff
@@ -193,6 +177,10 @@ Below you can find a list of all modules, for which at least one metric is defin
 Some extension modules expose histograms with timings of calls made to their backends.
 Please check the documentation of modules that are enabled in your config file, in order to learn if they provide them.
 
+All module backend metrics names use following convention: `[global, backends, Module, BackendAction]` and `[global, backends, Module, BackendAction, count]`.
```

Comment:
All module backend metrics names use the following convention: `[global, backends, Module, BackendAction]` and `[global, backends, Module, BackendAction, count]`.